wazo_phoned/plugins/gigaset/templates/gigaset_results.jinja

```diff
@@ -3,9 +3,14 @@
 <list response="get_list" type="pr" total="{{ total }}" first="{{ offset + 1 }}" last="{{ last }}">
 {%- for result in results %}
 <entry id="{{ result.number|e }}">
-{% set first_name, last_name = result.name.split(maxsplit=1) -%}
-<ln>{{ last_name|e }}</ln>
-<fn>{{ first_name|e }}</fn>
+{% set name_tuple = result.name.split(maxsplit=1) -%}
+{% if name_tuple|length < 2 -%}
+<ln></ln>
+<fn>{{ name_tuple[0] }}</fn>
```

Comment:
Escaping is missing here

Reply:
> Escaping is missing here

Thanks, it's fixed :smile: I amended my commit